### DISCUSSION TITLE
Wie korrekte Beta-Versionen testen?

### DISCUSSION THREAD

**Discussion by Erwin40** [Zefau/ioBroker.jarvis#2733] [Q&A]
Hallo zusammen.

Ich frage jetzt vielleicht etwas doof, aber wie genau muss man denn Beta-Versionen installieren, wenn man diese korrekt testen will?
Ausgehend mal von der installierten beta.49 gehe ich über Katze-Icon und dann auf den Tab "Benutzerdefiniert"
Dort gebe ich dann z,B. nachfolgende URL ein: https://github.com/Zefau/ioBroker.jarvis/tree/v3.2.0-beta.54
![grafik](https://github.com/user-attachments/assets/a17f13db-7917-4720-9941-be2b4c156643)

Ich beende Jarvis vorher nicht, deinstalliere die beta.49 nicht und boote den Raspi nicht neu.
Einzig was ich dann noch anschließend mache, ist auf den Geräten den Browser-Cache zu löschen.
Bei den iPads ist das ja der Verlauf (Tab Jarvis muss geschlossen sein) bei FireFox die Chronik:
Hier lösche ich nur "Temporäre Dateien und Seiten im Cache"
![grafik](https://github.com/user-attachments/assets/28415bde-39a8-4c27-9a37-6a3124ad5395)

**Comment by Bernd9000** [comment]
Du mußt bei "code" `commits` anklicken.
Dort in der Liste die Version raussuchen die Du haben möchtest und rechts den Doppelpfeil anklicken.
In der Browser Adresszeile ist dann der LInk, diesen bei der "Katze" "Benutzerdefiniert" eintragen

Der Link sieht dann z.B. so aus:
`https://github.com/Zefau/ioBroker.jarvis/tree/7492e53b87211c2b5dc032a9f6db3fd6d3e54d9e`


![Unbenannt2](https://github.com/user-attachments/assets/dc3e0ec4-0e94-4380-9844-52f537f6a9ee)

![Unbenannt](https://github.com/user-attachments/assets/01400147-26d2-400b-b51c-8577da2c7f2c)

**Comment by Bernd9000** [comment]
Ach ja....
Wenn Du nur die aktuelle letzte Version haben möchtest reicht folgender Link:
`https://github.com/Zefau/ioBroker.jarvis.git`

Nach der Installation:
Browser cache und eventuell auch cookies löschen.
IOB Neustart
